Formula/cmake.rb

```diff
@@ -27,6 +27,13 @@ class Cmake < Formula
   def install
     ENV.cxx11 if MacOS.version < :mavericks
 
+    # Avoid the following compiler error:
+    # SecKeychain.h:102:46: error: shift expression '(1853123693 << 8)' overflows
+    if MacOS.version <= :lion
+      ENV.append "CFLAGS", "-fpermissive"
+      ENV.append "CXXFLAGS", "-fpermissive"
+    end
```

Comment:
I suggest (for style)
```ruby
ENV.append_to_cflags "-fpermissive" if MacOS.version <= :lion
```

Reply:
Your suggested single line works for both `CFLAGS` and `CXXFLAGS`?

Reply:
Yes.
https://github.com/Homebrew/brew/blob/8778cc308ac6256c2e257962e1783d7f425ba6f1/Library/Homebrew/extend/ENV/shared.rb#L14
https://github.com/Homebrew/brew/blob/8778cc308ac6256c2e257962e1783d7f425ba6f1/Library/Homebrew/extend/ENV/shared.rb#L49-L51

Reply:
Thanks.  Change pushed.